https://github.com/ros-infrastructure/rospkg/pull/90

commit 7be8d9e5ec3c6bffd970e93053078b707879c7b9
Author: Alexis Ballier <aballier@gentoo.org>
Date:   Tue Oct 13 10:08:42 2015 +0200

    src/rospkg/os_detect.py: Use "string in dict" instead of "dict.has_key(string)" for python3 compatibility.

diff --git a/src/rospkg/os_detect.py b/src/rospkg/os_detect.py
index 14b57c2..554b607 100644
--- a/src/rospkg/os_detect.py
+++ b/src/rospkg/os_detect.py
@@ -150,7 +150,7 @@ class FdoDetect(OsDetector):
     """
     def __init__(self, fdo_id):
         release_info = read_os_release()
-        if release_info is not None and release_info.has_key("ID") and release_info["ID"] == fdo_id:
+        if release_info is not None and "ID" in release_info and release_info["ID"] == fdo_id:
             self.release_info = release_info
         else:
             self.release_info = None
